tools/amd_build/disabled_features.yaml

```diff
@@ -78,7 +72,9 @@
       {
         "path": "aten/src/THC/THCTensorRandom.cu",
         "s_constants": {
-          "struct curandStateMtgp32*": "curandStateMtgp32*"
+          "struct curandStateMtgp32*": "curandStateMtgp32*",
+          "__host__ void THCRandom_getRNGState": "extern \"C\" __host__ void THCRandom_getRNGState",
```

Comment:
Could you comment with an explanation for why this extra modification is necessary? The declarations of these functions already have "extern C" (via THC_API macro) inside aten/src/THC/THCTensorRandom.h. Thus these functions will already have support for linkage to modules written in the C programming language. It seems redundant to me.

The C++ standard:
_A function can be re-declared without a linkage specification after it was declared with a language specification, the second declaration will reuse the first language linkage._ 

https://en.cppreference.com/w/cpp/language/language_linkage

Reply:
As in the commit message: this is a workaround for a compiler frontend bug we filed already.

Reply:
Just saw, awesome!